demo/package.json

```diff
@@ -15,6 +15,7 @@
     "history": "^5.0.0",
     "react": "16.14.0",
     "react-dom": "16.14.0",
+    "react-github-btn": "1.2.1",
```

Comment:
Now I know there is everything to be found on npm. 😂

Reply:
lol yeah, for real. TBH I didn't have time to look into why the basic script tag method was working in local but not in PR demo env or demo site... but this works so 🤷‍♂️